docs/_Docs_GetStarted.md

```diff
@@ -35,7 +35,51 @@ Alternatively, you can start a local test node and deploy the contracts yourself
 ```
 ~$ yarn run start:blockchain:client
 
-~$ ./node_modules/.bin/truffle migrate
+~$ ./node_modules/.bin/truffle migrate --reset --compile-all
 ```
 
 For more detailed instructions, and additional steps required to set up an environment for use with colonyJS, refer to the [colonyJS get started doc](/colonyjs/docs-get-started/).
+
+## Set Up Reputation Mining for Local Testing
+
+The Reputation Mining client is usable for testing, but has a limited functionality. There is currently no challenge-response process to accommodate multiple submitted Reputation Root Hashes. Still, it is possible to run a single miner instance for usable reputation scores on a testnet.
+
+### Start the Mining Client
+
+Start the mining client with:
+
+```
+node packages/reputation-miner/bin/index.js --file ./reputations.json --colonyNetworkAddress 0x76d508fa65654654ffdb334a3023353587112e09 --minerAddress 0xb77d57f4959eafa0339424b83fcfaf9c15407461
+```
+
+This will use the first account in `ganache-accounts.json` as the Reputation miner.
+
+### Force Reputation Updates
+The client is set to provide a reputation update once per hour. For testing, you'll likely want to 'fast-forward' your network through a few submissions to see usable reputation.
+
+Move the network forward by an hour with:
+```
+$ curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_increaseTime","params":[3600],"id": 1}' localhost:8545
+```
+And mine a new block:
+```
+$ curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"evm_mine","params":[]}' localhost:8545
+```
+
+Note that because reputation is awarded for the *previous* submission window, you will need to use the 'fast-forward' commands above to speed through at least 3 reputation updates before noticing a change in the miner's reputation.
+
+### Get Reputation from the Reputation Oracle
+The mining client will answer queries for Reputation scores locally over HTTP:
+```
+http://127.0.0.1:3000/{colonyAddress}/{skillId}/{userAddress}
+```
+
+For example, you can get the reputation score of the miner using the address of the Meta Colony, the skill tag of `0`, and the address of the miner:
```

Comment:
Maybe add the addresses in parens after `Meta Colony` and `address of the miner`